misc/tremulous-grangerhub.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Tremulous Grangerhub
```

Comment:
Is the " Grangerhub" suffix in name really necessary if we already have it in the Comment section (and also in the Summary section of AppStream metadata file)?

We could optionally modify the icon file and add something like "GH" text to the corner to make the branding more apparent. But I do not think that changing the name of the actual game is a good idea. (Especially considering that the Grangerhub version is de facto the only version of Tremulous that is still used nowadays.)